src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -104,53 +99,26 @@ void Notifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const std::st
     std::string  uriPath;
     std::string  protocol;
 
-    if (!parseUrl(url, host, port, uriPath, protocol))
-    {
-      LM_E(("Runtime Error (not sending NotifyContextRequest: malformed URL: '%s')", url.c_str()));
-      return;
-    }
-
-    /* Set Content-Type */
-    std::string content_type = "application/json";
-
-#ifdef SEND_BLOCKING
```

Comment:
SEND_BLOCKING has not been in use sinde months (or years?). In addition, keeping this dead code aligned with the changes implemented in this PR for WS support will complicate things, so let's definetively remove it.